### PR TITLE
extend SignatureAlgorithms of sha256-rsa-MGF

### PR DESCRIPTION
**Pull request by dieHeike79**

### CODE REVIEW

File: src/ITfoxtec.Identity.Saml2/Schemas/Saml2SecurityAlgorithms.cs [hunk 1]
```diff
@@ -38,5 +38,6 @@ public static class Saml2SecurityAlgorithms
         /// </summary>
         public const string RsaSha512Signature = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha512";
```
Comment (fmaeseele) [inline src/ITfoxtec.Identity.Saml2/Schemas/Saml2SecurityAlgorithms.cs:40]: A comment like for the other constants would be nice

Comment (dieHeike79) [reply src/ITfoxtec.Identity.Saml2/Schemas/Saml2SecurityAlgorithms.cs]: added comment

**Comment by Revsgaard** [comment]
Thank you.

**Comment by Revsgaard** [comment]
Released in https://github.com/ITfoxtec/ITfoxtec.Identity.Saml2/releases/tag/4.8.7-beta1

**Pull request merged**